### ISSUE TITLE
(suggestion) Add gitter.im link to README

### ISSUE THREAD

**Issue by faceleg** [bower/bower#2049]
Similar to: https://github.com/angular/universal

**Comment by kunyan** [comment]
we moving to https://discordapp.com/, no longer needed

**Comment by paulohp** [comment]
@kunyan we still have a public bower chat there. gitter.im/bower/bower

**Comment by faceleg** [comment]
Discord is blocked by my corporate proxy
